src/main/resources/Jetbrains_Dark.theme.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
have you tested these themes ? are they looking good ? could you please share screenshots ?

Reply:
screenshots for all three variants

Reply:
I have tested these three themes :
1 - Dark mood 
It is looking as expected.

2- Light mood
It is looking as expected
![Dark](https://user-images.githubusercontent.com/88733573/130599113-9feda51a-4217-4a8f-be96-0977bc61a9b6.PNG)
![Gray](https://user-images.githubusercontent.com/88733573/130599121-cfa7b41e-c47a-44b6-ba83-7030dec24557.PNG)
![Light](https://user-images.githubusercontent.com/88733573/130599124-38ed6f39-32d4-4b39-8095-536e2c33a502.PNG)

3- Dark mood
I am still working on it.

Please find the attached screenshot for your reference